src/components/Step.vue

```diff
@@ -124,7 +130,14 @@ export default class Step extends Vue {
     | undefined;
 
   get stepName(): string {
-    return humanReadableLabel(this.step);
+    if (this.step.name === 'domain') {
+      // enable to retrieve the related name of a query referenced behind an uid
+      return humanReadableLabel(this.step, (domain: string | ReferenceToExternalQuery) =>
+        retrieveDomainName(domain, this.queries),
+      );
+    } else {
+      return humanReadableLabel(this.step);
```

Comment:
Maybe that function could be passed to all `humanReadableLabel`?
Even if it's not used, it shouldn't be a problem

Reply:
And it makes one less if :D

Reply:
je fais la modif :pray:

Reply:
https://github.com/ToucanToco/weaverbird/pull/1437/commits/2e058ec4390ca176a5376c4d0ad76f72ff9d33b7